### PR TITLE
[cumulus] Improved check for sane bridge fees calculations

### PR DESCRIPTION
**Pull request by bkontur**
## TODO
- [x] change constants when CI fails (should fail :) )

## Result

On the AssetHubRococo: 1701175800126 -> 1700929825257 = 0.15 % decreased.
```
# Before ( [xcm] Fix `SovereignPaidRemoteExporter` and `DepositAsset` handling (#3157))
Feb 02 12:59:05.520 ERROR bridges::estimate: `bridging::XcmBridgeHubRouterBaseFee` actual value: 1701175800126 for runtime: statemine-1006000 (statemine-0.tx14.au1)    

# After
Feb 02 13:02:40.647 ERROR bridges::estimate: `bridging::XcmBridgeHubRouterBaseFee` actual value: 1700929825257 for runtime: statemine-1006000 (statemine-0.tx14.au1)    

```

On the AssetHubWestend:  2116038876326 -> 1641718372993 = 22.4 % decreased.
```
# Before ( [xcm] Fix `SovereignPaidRemoteExporter` and `DepositAsset` handling (#3157))
Feb 02 12:56:00.880 ERROR bridges::estimate: `bridging::XcmBridgeHubRouterBaseFee` actual value: 2116038876326 for runtime: westmint-1006000 (westmint-0.tx14.au1)    


# After
Feb 02 13:04:42.515 ERROR bridges::estimate: `bridging::XcmBridgeHubRouterBaseFee` actual value: 1641718372993 for runtime: westmint-1006000 (westmint-0.tx14.au1)    
```

### CODE REVIEW

File: cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs [hunk 1]
```diff
@@ -21,3 +21,60 @@ pub mod test_data;
...
+	sp_tracing::try_init_simple();
+	log::error!(
+		target: "bridges::estimate",
+		"{label}:\nconstant: {const_name}\n[+] actual: {actual}\n[+] estimated: {estimated} ({diff_to_estimated:.2?})\n[+] estimated(+33%): {estimated_plus_overestimate} ({diff_to_estimated_plus_overestimate:.2?})",
+	);
+
+	// check if estimated value is sane
+	assert!(
+		estimated <= actual,
+		"estimated: {estimated}, actual: {actual}, please adjust `{const_name}` value",
```
Comment (svyatonik) [inline cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs:52]: I suppose we need to change to `estimated_plus_overestimate` here. Maybe add it to the log? Something like: "please adjust `{const_name}` value to `{estimated_plus_overestimate}`"?

File: cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs [hunk 2]
```diff
@@ -21,3 +21,60 @@ pub mod test_data;
...
+	);
+	assert!(
+        estimated_plus_overestimate <= actual,
+        "estimated_plus_overestimate: {estimated_plus_overestimate}, actual: {actual}, please adjust `{const_name}` value",
```
Comment (svyatonik) [inline cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs:56]: Also maybe better to be specific here and specify that it needs to be changed to `estimated_plus_overestimate`?

File: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs [hunk 3]
```diff
@@ -594,55 +602,43 @@ mod bridge_hub_bulletin_tests {
...
+		bridge_hub_test_utils::check_sane_fees_values(
+			"bp_bridge_hub_rococo::BridgeHubRococoBaseDeliveryFeeInRocs",
+			bp_bridge_hub_rococo::BridgeHubRococoBaseDeliveryFeeInRocs::get(),
+			|| {
+				from_grandpa_chain::can_calculate_fee_for_complex_message_delivery_transaction::<
+					RuntimeTestsAdapter,
+				>(collator_session_keys(), construct_and_estimate_extrinsic_fee)
+			},
+			Perbill::from_percent(33),
+			None, /* we don't want lowering according to the Bulletin setup, because
```
Comment (svyatonik) [inline cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs:616]: :+1:

**Review by svyatonik — APPROVED** [review]
LGTM

File: cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs (hunk 1 above)
Comment (bkontur) [reply cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs]: thank you, good point, fixed

File: cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs (hunk 2 above)
Comment (bkontur) [reply cumulus/parachains/runtimes/bridge-hubs/test-utils/src/lib.rs]: also fixed

**Review by acatangiu — APPROVED** [review]

**Pull request merged**